api-interop-layer/data/index.js

```diff
@@ -0,0 +1,110 @@
+import getAlerts from "./alerts/index.js";
+import getForecast from "./forecast/index.js";
+import getObservations from "./obs/index.js";
+import getPoint from "./points.js";
+import getSatellite from "./satellite.js";
+import dayjs from "../util/day.js";
+
+export const getDataForPoint = async (lat, lon) => {
+  const { point, place, grid } = await getPoint(lat, lon);
+
+  const satellitePromise = getSatellite({ grid });
+
+  const { forecast, observed } = await Promise.all([
+    getForecast({ grid }),
+    getObservations({ grid, point }),
+  ]).then(([forecastData, obsData]) => {
```

Comment:
For now, the wrapper around `fetch` re-throws. Because it's not handled here, it bubbles up to the fastify default error handler (defined [here](https://github.com/weather-gov/weather.gov/pull/1590/files#diff-28a897d8f1355182563a7be7f7191e9e0fe812969cb9d0276404ce865f1be1f8R9)), that console logs out error and returns a 500 error. That's just there as a failsafe to guard against otherwise-handled exceptions. It really shouldn't be the general-purpose error handler. 😬